spec/controllers/synapses_controller_spec.rb

```diff
@@ -35,13 +35,30 @@
       end
     end
 
+    context 'with private topics' do
+      it 'redirects to /' do
```

Comment:
Because of how we handle Pundit::NotAuthorized in application controller - we don't return 403, we do conditional redirects. I was surprised too, but it is how it works so the test is at least documenting that!